frontend/src/component/MeetupForm/MeetupForm.tsx

```diff
@@ -120,7 +120,11 @@ export const MeetupForm: FunctionComponent = () => {
                   type="submit"
                   disabled={isSubmitting || isLoading || isSuccess}
                 >
-                  Next
+                  {isSubmitting || isLoading || isSuccess ? (
```

Comment:
why do we put `isSuccess` here?

Reply:
I was following this.
Because, I don't want the user to press the submit button again after submitting the form.

disabled={isSubmitting || isLoading || isSuccess}

Reply:
They don't need to see the loading state once form isSuccess though so you can remove that.

Reply:
removed